src/Core/Session.php

```diff
@@ -104,117 +102,11 @@ public static function remove($name)
 	 */
 	public static function clear()
 	{
+		session_unset();
```

Comment:
What is the point of adding these two lines?

Reply:
I'm not sure about backwards compatibility in combination with `session_start()`, so I decided to use some older functions as well. Do you think this isn't necessary?

Reply:
There seem to be a `destroy` method that does the `session_unset` (but not the `session_start`) so I was wondering if you have a specific reason for adding these two lines here. I don't think it is necessary,